http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -292,30 +290,30 @@ STEP_TIMINGS+=($(date +%s))
 
 # Jupyter-specific setup, only do if Jupyter is installed
 if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
-  log 'Installing Jupydocker kernelspecs...'
-
   # user package installation directory
   mkdir -p /work/packages
   chmod a+rwx /work/packages
 
   # Used to pip install packacges
+  JUPYTER_USER_PIP_DIR=$JUPYTER_USER_HOME/.local/lib/python3.7/site-packages
+
   # TODO: update this if we upgrade python version
-  ROOT_USER_PIP_DIR=/usr/local/lib/python3.7/dist-packages
-  JUPYTER_USER_PIP_DIR=/home/jupyter-user/.local/lib/python3.7/site-packages
-
-  # Install kernelspecs inside the Jupyter container
-  # TODO This is baked into terra-jupyter-base as of version 0.0.6. Keeping it here for now to support prior image versions.
-  retry 3 docker exec -u root ${JUPYTER_SERVER_NAME} ${JUPYTER_SCRIPTS}/kernel/kernelspec.sh ${JUPYTER_SCRIPTS}/kernel ${KERNELSPEC_HOME}
-
-  # Install jupyter_notebook_config.py
-  # TODO This is baked into terra-jupyter-base as of version 0.0.6. Keeping it here for now to support prior image versions.
-  if [ ! -z "$JUPYTER_NOTEBOOK_CONFIG_URI" ] ; then
-    log 'Copy Jupyter notebook config...'
-    gsutil cp ${JUPYTER_NOTEBOOK_CONFIG_URI} /etc
-    JUPYTER_NOTEBOOK_CONFIG=`basename ${JUPYTER_NOTEBOOK_CONFIG_URI}`
-    docker cp /etc/${JUPYTER_NOTEBOOK_CONFIG} ${JUPYTER_SERVER_NAME}:${JUPYTER_HOME}/
+  if [ "$JUPYTER_USER_HOME" = "/home/jupyter" ]
+  then
+     ROOT_USER_PIP_DIR=/opt/conda/lib/python3.7/site-packages
```

Comment:
where did we land on supporting conda?

Reply:
We are definitely using conda...there's no point not using it since the deep learning image comes with it...the question is whether we want to enable conda tab in UI....for that, what I was thinking is having this PR merged, and ask someone to try out the new terra-docker images to see if they think it's worth enabling conda tab.

This PR does not enable or disable conda tab.